application/comit_node/src/swap_protocols/rfc003/ethereum/contract_templates/erc20_contract.asm

```diff
@@ -84,7 +84,7 @@ exit:
 
 */
 redeem:
-    log1(0, 0, 0xB8CAC300E37F03AD332E581DEA21B2F0B84EAAADC184A295FEF71E81F44A7413) // log keccak256(Redeemed())
+    log1(0, 32, 0xB8CAC300E37F03AD332E581DEA21B2F0B84EAAADC184A295FEF71E81F44A7413) // log keccak256(Redeemed())
```

Comment:
:wrench: Modify the comment to state that the log includes the secret and where?